### ISSUE TITLE
Support for testing HTTP PUT and POST

### ISSUE THREAD

**Issue by iyesin** [syepes/network_exporter#28]
Hello,

I believe it would be nice to have a way to submit pseudo-random buffer (or user-specified static file) to user-specified URLs and measure same set parameters as for GET, plus `http_put_seconds{type=ContentTransfer}` (keeping `http_get_seconds` as is for compatibility reasons). WDYT?

**Comment by syepes** [comment]
Yes this woudl be an interesting feature, the only thing I see is how and what format/type of data we shoud send but specially what backends/enpoints could be supported for the reception of the payload.

I would be open for PR :-)
